test/filters/oidc/jwks_storage_test.cc

```diff
@@ -0,0 +1,69 @@
+#include "src/filters/oidc/jwks_storage.h"
+
+#include "gtest/gtest.h"
+
+namespace authservice {
+namespace filters {
+namespace oidc {
+
+namespace {
+// A good public key based on above private key
```

Comment:
not sure i follow this comments. where is the "above private key"? this is at the top of the file.